15 - LocalStorage/index-START.html

```diff
@@ -26,13 +26,58 @@ <h2>LOCAL TAPAS</h2>
   </div>
 
 <script>
-  const addItems = document.querySelector('.add-items');
-  const itemsList = document.querySelector('.plates');
-  const items = [];
+  class customStoredList {
+    constructor(wrapperClass) {
+      this.wrapper = document.querySelector(wrapperClass);
+      this.addItems = this.wrapper.querySelector('.add-items');
```

Comment:
nice scoping, so you do not end up with the wrong element.

Reply:
😄